src/ralph/conf.py

```diff
@@ -262,6 +262,8 @@ class Config(BaseSettingsConfig):
 
     _CORE: CoreSettings = core_settings
     AUTH_FILE: Path = _CORE.APP_DIR / "auth.json"
+    AUTH_PROVIDER_URL: str = "http://localhost:8081/auth/realms/ralph"
```

Comment:
I think we can be more pydantic on this :wink: https://docs.pydantic.dev/usage/types/#urls